app/_src/mesh/features/meshopa.md

```diff
@@ -0,0 +1,668 @@
+---
+title: MeshOPA (beta) - OPA Policy Integration
+---
+
+{% warning %}
```

Comment:
This warning is not visible on the website. Also, similar warnings for Kuma policies are not rendered in Kong docs. This is unfortunate because Kong Mesh users can be unaware that new policies cannot be mixed with old ones and that they could change before getting out of beta state

Reply:
It is visible, it's just not styled correctly - 

![image](https://user-images.githubusercontent.com/2753650/215037955-3a0309ce-490b-4342-b777-da39794193ed.png)

same with MeshTrace - 
![image](https://user-images.githubusercontent.com/2753650/215038041-d1ee4963-63ac-49a8-ae11-4c916a67527e.png)

I'm guessing this needs to be unified so it works both in KM and Kuma - @fabianrbz any ideas on how to fix it?

Reply:
Kuma for reference:

![image](https://user-images.githubusercontent.com/2753650/215038571-1a2a786d-3155-4c03-97ab-afc370956702.png)

Reply:
I created a jira issue for this, look in slack for my thread. Closing this for now.